autogpt/commands/file_operations.py

```diff
@@ -228,22 +228,6 @@ def write_to_file(filename: str, text: str, agent: Agent) -> str:
         return f"Error: {err}"
 
 
-@command(
-    "append_to_file",
-    "Appends to a file",
-    {
-        "filename": {
-            "type": "string",
-            "description": "The name of the file to write to",
-            "required": True,
-        },
-        "text": {
-            "type": "string",
-            "description": "The text to write to the file",
-            "required": True,
-        },
-    },
-)
 @sanitize_path_arg("filename")
 def append_to_file(
```

Comment:
Should we not be removing unused code? I will be still in the commit history if we want to get it back

Reply:
it's still used by the file logger